### PR TITLE
20180910 1228

### PR DESCRIPTION
**Pull request by farhanpatel**

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -322,6 +322,10 @@
         <source>This action will clear all of your private data. It cannot be undone.</source>
         <note>Description of the confirmation dialog shown when a user tries to clear their private data.</note>
       </trans-unit>
+      <trans-unit id="This action will clear all of your website data. It cannot be undone.">
```
Comment (flodolo) [inline templates/firefox-ios.xliff]: This should an ID, not the message itself.

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -2006,6 +2026,17 @@
         <source>Troubleshoot</source>
         <note>Title of link to help page to find out how to solve Sync issues</note>
       </trans-unit>
+      <trans-unit id="Settings.WebsiteData.Button">
+        <source>Show More</source>
+        <note>Button shows all websites on website data tableview
```
Comment (flodolo) [inline templates/firefox-ios.xliff]: Any chance to fix these localization comments, and have only one?

File: templates/firefox-ios.xliff [hunk 3]
```diff
@@ -2751,6 +2782,25 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Shared/Supporting Files/firefoxcredentials.strings" source-language="en" datatype="plaintext">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
+    </header>
+    <body>
+      <trans-unit id="Available Credentials:">
```
Comment (flodolo) [inline templates/firefox-ios.xliff]: Need ID

File: templates/firefox-ios.xliff [hunk 4]
```diff
@@ -2751,6 +2782,25 @@
...
+        <source>Available Credentials:</source>
+        <note>Header for the list of credentials table</note>
+      </trans-unit>
+      <trans-unit id="Firefox Credentials">
```
Comment (flodolo) [inline templates/firefox-ios.xliff]: Need ID

File: templates/firefox-ios.xliff [hunk 5]
```diff
@@ -2751,6 +2782,25 @@
...
+        <source>Firefox Credentials</source>
+        <note>Title of the extension that shows firefox passwords</note>
+      </trans-unit>
+      <trans-unit id="You don’t have any credentials synced from your Firefox Account">
```
Comment (flodolo) [inline templates/firefox-ios.xliff]: Need ID

**Comment by farhanpatel** [comment]
Thanks for the feedback @flodolo opened a new PR

**Pull request closed without merging**